assets/server/login/verify-email-check.html

```diff
@@ -31,32 +38,28 @@
 
   <script defer type="text/javascript">
     window.addEventListener('load', (event) => {
+      let verifyPending = document.querySelector('span#verify-pending');
+      let verifyError = document.querySelector('span#verify-error');
+      let showError = () => {
+        verifyPending.classList.add('d-none');
+        verifyError.classList.remove('d-none');
+      }
+
       let urlVars = getUrlVars();
-      let code = urlVars["oobCode"];
+      let code = urlVars['oobCode'];
       if (!code) {
-        code = "";
+        showError();
+        return;
       }
 
       firebase.auth().applyActionCode(code)
         .then(function(resp) {
-          window.location.assign("/");
-        }).catch(function(error) {
-          flash.clear();
-          flash.error("Invalid email verification code. "
```

Comment:
No flash?

Reply:
It's rendered in HTML on L28. Go's templating language doesn't work when you try to embed the translated string into a javascript string.